plugins/greynoise/unit_test/test_community_lookup.py

```diff
@@ -0,0 +1,55 @@
+import sys
+import os
+
+sys.path.append(os.path.abspath("../"))
+
+from unittest import TestCase, mock
+from icon_greynoise.connection.connection import Connection
+from icon_greynoise.actions.community_lookup import CommunityLookup
+from greynoise import GreyNoise
+import json
+import logging
+
+
+def read_file_to_string(filename):
+    with open(filename) as my_file:
+        return my_file.read()
+
+
+def mocked_requests_get(*args, **kwargs):
+    actual_path = os.path.dirname(os.path.realpath(__file__))
+    actual_joined_path = os.path.join(actual_path, "payloads/community_ip.json")
+    get_messages_from_user_payload = read_file_to_string(actual_joined_path)
+    return json.loads(get_messages_from_user_payload)
+
+
+class MockConnection:
+    def __init__(self):
+        self.server = "test_server"
+        self.api_key = "test_api_key"
+        self.user_agent = "test_user_agent"
```

Comment:
I think this could be extracted to separate file as it's being copy/pasted along all the tests